zoo-css-challenge/style.css

```
@@ -188,6 +238,11 @@ footer {
   color: #ff7f11;
 }
 
+#member-container {
+  margin-left: 250px;
+  margin-right: 250px;
```

```suggestion
  margin: auto;
```
Alternatively, we can use `margin: auto` which centres the element. `auto` distributes the left and right margins equally. We can also move these properties into the `.membership` rule above on line 227 as mentioned in my other comment